src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -58,6 +58,25 @@
                     </include>
                 </foreach>
             </if>
+            <if test="studyViewFilter.mutationDataFilters != null and !studyViewFilter.mutationDataFilters.isEmpty()">
```

Comment:
lets put some comments in this which explain how the AND/OR logic works.

Reply:
This part is not needed at this moment. I commented it out and will check on this when we implement the second half of the endpoint